docs/pages/access-controls/guides/access-requests-slack.mdx

```diff
@@ -0,0 +1,347 @@
+
+Then scroll to "Scopes", and add the following scopes so the plugin can post messages to your Slack channels:
+
+- `chat:write`
+- `incoming-webhook` 
+- `users:read` 
+- `users:read.email` 
+
+![API Scopes](../../../img/enterprise/plugins/slack/api-scopes.png)
+
+#### Obtain an OAuth token
+
+Slack will generate an OAuth token for you automatically. Find it at the top under the "OAuth Tokens for Your Workspace" section:
+
+![OAuth Tokens](../../../img/enterprise/plugins/slack/OAuth.png)
+
+#### Add the app to your Slack Workspace
+
+Add the app to your Slack Workspace:
+
+![OAuth Tokens](../../../img/enterprise/plugins/slack/Slackbot-Permissions.png)
+
+After doing so, make sure to invite your bot to the channel. Do this by using the `@` command.
+
+![Invite bot to channel](../../../img/enterprise/plugins/slack/invite-user-to-team.png)
+
+## Step 3/4. Installing the Teleport Slack Plugin
+
+Since Teleport plugins typically augment your Teleport cluster with third-party extensions and integrations with minimal additional overhead, you'll want to consider your overall architecture. We recommend installing the Teleport Slack plugin alongside Teleport Proxy.
+
+We currently only provide `linux-amd64` binaries. You can optionally compile these plugins from [source](https://github.com/gravitational/teleport-plugins/tree/master/access/slack).
+
+<Tabs>
+  <TabItem label="Download">
+    ```bash
+    curl -LO https://get.gravitational.com/teleport-access-slack-v(=teleport.version=)-linux-amd64-bin.tar.gz
+    tar -xzf teleport-access-slack-v(=teleport.version=)-linux-amd64-bin.tar.gz
+    cd teleport-access-slack
+    ./install
+    ```
+
+    ```bash
+    # Output
+    Teleport Slack binaries have been copied to /usr/local/bin
+    You can run teleport-slack configure > /etc/teleport-slack.toml to bootstrap your config file
+    ```
+  </TabItem>
+  <TabItem label="From Source">
+    To install from source you need `git` and `go >= (=teleport.golang=)` installed.
+
+    ```bash
+    # Checkout teleport-plugins
+    git clone https://github.com/gravitational/teleport-plugins.git
+    cd teleport-plugins/access/slack
+    make
+    ```
+  </TabItem>
+</Tabs>
+
+<Admonition type="tip">
+  View the [Slack Plugin README](https://github.com/gravitational/teleport-plugins/tree/master/access/slack).
+</Admonition>
+
+### Configuring Teleport Slack
+
+Teleport Slack uses a config file in `TOML` format.
+
+Generate a boilerplate config by running the following command:
+
+```bash
+teleport-slack configure > teleport-slack.toml
+```
+
+#### Copy certificates
+
+Navigate back to the certificates you generated with `tctl auth sign` before. The plugin installer creates a folder for them at: `/var/lib/teleport/plugins/slack/`. 
+
+Copy or move the certificates there and point the config file to them:
+
+```bash
+# Navigate up directories
+cd ../
+# Move certificate files created by 'tctl auth sign'
+sudo mv -f auth.cas /var/lib/teleport/plugins/slack/
```

Comment:
Above, we switched to sign --format=file, so this would be just one file `mv -f auth /var/lib/teleport/plugins/slack`

Reply:
Done!